packages/verbs-sdk/src/verbs.ts

```diff
@@ -0,0 +1,44 @@
+import { PrivyWalletProvider } from './adapters/privy.js'
+import type { VerbsConfig, VerbsInterface } from './types/verbs.js'
+import type { WalletProvider } from './types/wallet.js'
+import type { Wallet } from './wallet.js'
+
+/**
+ * Main Verbs SDK class
+ * @description Core implementation of the Verbs SDK
+ */
+export class Verbs implements VerbsInterface {
+  createWallet!: () => Promise<Wallet>
+  getWallet!: (walletId: string) => Promise<Wallet | null>
+
+  private walletProvider: WalletProvider
+
+  constructor(config: VerbsConfig) {
+    this.walletProvider = this.createWalletProvider(config)
+
+    // Delegate wallet methods to wallet provider
+    this.createWallet = this.walletProvider.createWallet.bind(this.walletProvider)
+    this.getWallet = this.walletProvider.getWallet.bind(this.walletProvider)
+  }
```

Comment:
🎨 We may decide this is over-engineered, but my intention is to establish this pattern: The Verbs SDK delegates functionality to various provider abstractions, which may have various implementations.

Reply:
I think this makes sense. we can revisit when we design the MVP though!